odl/contrib/datasets/images/cambridge.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Images provided by the University of Cambridge."""
+
+import numpy as np
+from odl.contrib.datasets.util import get_data
+
+
+__all__ = ('brain_phantom', 'resolution_phantom', 'cms', 'cms_gray', 'rings',
+           'blurring_kernel')
+
+
+DATA_SUBSET = 'images'
+URL_CAM = 'http://store.maths.cam.ac.uk/DAMTP/me404/data_sets/'
+
+
+def brain_phantom():
+    """Brain phantom for FDG PET simulations.
+
+    The image is scaled between [0, 1].
+    """
+    # TODO: Store data in some ODL controlled url
+    name = 'PET_phantom.mat'
+    url = URL_CAM + name
+    dct = get_data(name, subset='images', url=url)
+    im = np.rot90(dct['im'].astype('float64'), k=3)
+    im /= im.max()
+
+    return im
+
+
+def resolution_phantom():
```

Comment:
This could actually be implemented using the `ellipse_phantom` stuff in ODL, which should give a variable resolution. I'd personally prefer doing ti that way.

Reply:
Agreed. How long would it take to do this? Maybe I can write a todo note there about this and we can implement this at some other point.